server/models/todo.ts

```diff
@@ -0,0 +1,24 @@
+import mongoose, { Model, Schema } from "mongoose";
+
+type TodoPriority = "High" | "Medium" | "Low";
+
+interface ITodo {
+  title: String;
+  priority: TodoPriority;
+  description?: String;
+  isChecked: boolean;
+  date: String;
```

Comment:
shouldn't the date type be Date ?

Reply:
In the front-end of the application, the values of the date are set via an input field with the type of date wich gets the date in the format of dd-mm-yyyy in a string, so thats why im storing it as a string.

Reply:
use Date here as type